cron/add_publish_date.py

```diff
@@ -0,0 +1,32 @@
+from django.db.models import Q
```

Comment:
Move this to management command. e.g. https://github.com/Spoken-tutorial/spoken-website/blob/eventsV2/base/management/commands/initial_data.py

Or just have it run once on the production server inside django shell.